### PR TITLE
Update SIG-OpenStack leadership

### PR DESCRIPTION
**Pull request by hogepodge**
SIG-OpenStack is undergoing leadership changes. We are seeking new volunteers to co-lead, and will send a second update once the volunteering process is complete.

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1448,12 +1448,6 @@ sigs:
       - name: Chris Hoge
         github: hogepodge
         company: OpenStack Foundation
-      - name: David Lyle
```
Comment (nikhita) [inline sigs.yaml]: Please update the README using `make generate`.

You can also list previous leads as `emeritus_leads` :)

https://github.com/kubernetes/community/blob/08bffde59eab640d3d404029b7f296931a084862/sigs.yaml#L979-L982

**Comment by dims** [comment]
cc @chaosaffe @chrigl @lingxiankong @Lion-Wei @adisky @zetaab @FengyunPan2 @NickrenREN @Fedosin

**Comment by hogepodge** [comment]
To all SIG-OpenStack members cc'ed on this, please see this mailing list post. https://groups.google.com/d/msg/kubernetes-sig-openstack/5HT-TYOuwrs/c2Q-3uIfEwAJ

**Comment by adisky** [comment]
lgtm

**Comment by lingxiankong** [comment]
all good

**Pull request merged**